app/src/main/res/layout/item_grid.xml

```diff
@@ -36,13 +37,19 @@
             android:paddingLeft="16dp"
             android:paddingRight="16dp">
 
-            <TextView
-                android:id="@+id/title"
+            <com.kabouzeid.gramophone.views.TouchInterceptHorizontalScrollView
```

Comment:
I'm not a fan of having this in the grid. It's a bit janky when you sometimes try to swipe to a different tab but end up scrolling a name in a small grid item instead. The user can always open it up and scroll it there anyway. :)